src/java.base/share/classes/java/util/Arrays.java

```diff
@@ -78,6 +78,26 @@ public final class Arrays {
     // Suppresses default constructor, ensuring non-instantiability.
     private Arrays() {}
 
+    @IntrinsicCandidate
+    private static void arraySort(int[] array, int fromIndex, int toIndex) {
```

Comment:
A minor styling comment: We can use same all small caps naming convention as used for System.arraycopy.

Reply:
Thanks for the suggestion. For now, we will stick with arraySort as this is a private method.